<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commentForm</class>
 <widget class="QWidget" name="commentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>81</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>656</width>
    <height>81</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lb_comment">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>33</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
background-color: rgb(202, 202, 255);</string>
   </property>
   <property name="text">
    <string>我也觉得！！--------------------------------------------------------</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>2</y>
     <width>649</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>31</height>
    </size>
   </property>
   <widget class="QLabel" name="lb_time">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 127);</string>
    </property>
    <property name="text">
     <string>时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Agency FB&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;昵称&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
